internal/common/common.go

```diff
@@ -166,6 +171,20 @@ func IsSingleNodeCluster(cluster *Cluster) bool {
 	return swag.StringValue(cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone
 }
 
+func GetClusterDeploymentMode(cluster *Cluster) string {
```

Comment:
We already have these properties for these, not according to the number of hosts.
SNO would be cluster.highAvailabilityMode == "None"
You actually don't care about compact, you want to know if cluster.SchedulableMasters is true.

Reply:
the Idea here is for  what type of cluster,
this `cluster.SchedulableMasters== true` applies to SNO and compact but I still need to check how many nodes or do  `cluster.SchedulableMasters== true && cluster.highAvailabilityMode == "None"` 

this simplify the check which used in LVM and ODF (  may be in other places )